packages/cli/src/Db.ts

```diff
@@ -1,6 +1,8 @@
 /* eslint-disable @typescript-eslint/restrict-template-expressions */
```

Comment:
we should make this file also into a `@Service`, and move it inside `src/databases`.